src/search/FindInFilesUI.js

```diff
@@ -102,10 +109,45 @@ define(function (require, exports, module) {
 
                 StatusBar.hideBusyIndicator();
             })
+            .progress(function (results) {
+                // Update first page of panel (100 matches) for every file with matches,
+                // then only update for every page (100 matches)
+                if (results.matches && results.matches.length) {
+                    progressCount += results.matches.length;
+                    if (!progressFirstUpdate) {
+                        // First update
+                        _resultsView.open();
+                        if (_findBar) {
+                            _findBar.close();
+                        }
+                        progressFirstUpdate = true;
+                    } else if (!progressFirstPage) {
+                        // First page
```

Comment:
100 should be defined in a constant (e.g. ENTRIES_PER_PAGE) if it's not already defined somewhere.